TaskLayer/SearchTask/PostSearchAnalysisTask.cs

```diff
@@ -607,7 +607,7 @@ private void WriteProteinResults()
                 // write all individual file results to subdirectory
                 // local protein fdr, global parsimony, global psm fdr
 
-                if ((Parameters.CurrentRawFileList.Count > 1 && Parameters.SearchParameters.WriteIndividualFiles ) 
+                if (Parameters.SearchParameters.WriteIndividualFiles
                     || Parameters.SearchParameters.WriteMzId || Parameters.SearchParameters.WritePepXml)
                 {
                     Directory.CreateDirectory(Parameters.IndividualResultsOutputFolder);
```

Comment:
I don't think we should make this folder if we only have one file. The mzid should instead be put in the parent directory with all the other files.